packages/ui-library/src/components/select-field/stories.js

```diff
@@ -11,20 +11,33 @@ export default {
 	parameters: {
 		docs: {
 			description: {
-				component: "A simple select field component.",
+				component: "A simple select field component. Any extra props are passed on to the inner `Select` element, please refer to its own documentation.",
 			},
 		},
 	},
 };
 
+const options = [
+	{ value: "1", label: "Option 1" },
+	{ value: "2", label: "Option 2" },
+	{ value: "3", label: "Option 3" },
+	{ value: "4", label: "Option 4" },
+];
+
 const Template = ( args ) => {
-	const [ value, setValues ] = useState( args.value || "" );
-	const handleChange = useCallback( setValues, [ setValues ] );
+	const [ value, setValue ] = useState( args.value || "" );
+	const [ selectedLabel, setSelectedLabel ] = useState( value ? options.find( option => option.value === value ).label : "" );
```

Comment:
Shouldn't `options` go via the `args`?

Reply:
I changed it to args.options and added the options to the default args.